### DISCUSSION TITLE
synology Docker

### DISCUSSION THREAD

**Discussion by s128300901** [v2rayA/v2rayA#1590] [Q&A]
tell me, when you turn on the transparent proxy, an error occurs failed to start v2ray-core: not support "redirect" mode of transparent proxy: ExecCommands: iptables-legacy -w 2 -t nat -N TP_OUT iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root) : exit status 4

**Comment by s128300901** [comment]
![Без имени](https://github.com/user-attachments/assets/debac974-9783-4f91-be21-09a67084764b)

**Comment by bzany** [comment]
![屏幕截图 2025-01-02 192912](https://github.com/user-attachments/assets/de5d5ade-1226-4221-b514-513ca74035d5)

**Reply by bzany** [reply]
你不给他root权限怎么运行
